packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -167,6 +179,40 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+export interface CompatVersion {
+	base: string;
+	delta: number;
+}
+
+export interface CompatVersionConfig {
+	name: string;
+	createWith: CompatVersion;
+	loadWith: CompatVersion;
+}
```

Comment:
"Compat" is too overloaded a term. Maybe CrossMajorVersionCompat?

Reply:
I agree. Given that now "compat" can mean many things, it may be worthwhile considering how we get "compat" terminology altogether. In essence, find a way for a test config to be more explicit on what versions we want to use for load vs create, and list out variations to be considered.